### PR TITLE
Issue 2 view entries - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
https://user-images.githubusercontent.com/122064415/218897152-ea0997de-e0da-4e34-a3f1-ebfea774abb5.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -20,6 +20,83 @@ function save(event) {
...
+  if ($noEntry.className === 'column-full no-entry') {
+    toggleNoEntries();
+  }
 }
 
 $journal.addEventListener('submit', save);
+
+function renderEntry(entry) {
+  var $entryList = document.createElement('li');
+  document.querySelector('ul').appendChild($entryList);
```
Comment (shawnkost) [inline js/main.js]: You shouldn't be appending to the `ul` here, this function should only be responsible for creating the DOM tree for this entry. The actual appending to the "entry-list" should happen outside of this function, wherever you call it

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Justin!

**Comment by justinsong1608** [comment]
Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**